Library/Homebrew/download_strategy.rb

```diff
@@ -1404,18 +1404,18 @@ def self.detect_from_url(url)
     when %r{^https?://www\.apache\.org/dyn/closer\.cgi},
          %r{^https?://www\.apache\.org/dyn/closer\.lua}
       CurlApacheMirrorDownloadStrategy
-    when %r{^https?://(.+?\.)?googlecode\.com/svn},
+    when %r{^https?://([A-Za-z0-9\-.]+\.)?googlecode\.com/svn},
```

Comment:
```suggestion
    when %r{^https?://(\w\-.]+\.)?googlecode\.com/svn},
```

maybe and elsewhere?

Reply:
`\w` includes `A-Za-z0-9` but also `_` which [isn't valid in a hostname](https://man7.org/linux/man-pages/man7/hostname.7.html).

Reply:
TIL!